dockerGrader/test/p3/test.py

```diff
@@ -0,0 +1,260 @@
+#!/usr/bin/python
```

Comment:
Do we plan to check identical test.py files in?  Why not refer to the actual one (which is also checked in with this repo)

Reply:
Sounds good. I think you may update the test.py after the first time I downloaded.